.github/workflows/build.yml

```diff
@@ -178,10 +182,27 @@ jobs:
       - name: "Install dependencies"
         run: "composer install --no-interaction --no-progress"
 
-      - uses: "actions/download-artifact@v4"
+      - name: "Checkout build-infection"
+        uses: actions/checkout@v5
         with:
-          name: "result-cache-${{ matrix.php-version }}"
-          path: "tmp/"
+          repository: "phpstan/build-infection"
+          path: "build-infection"
+          ref: "1.x"
+
+      - name: "Install build-infection dependencies"
+        working-directory: "build-infection"
+        run: "composer install --no-interaction --no-progress"
+
+      - name: "Configure mutators"
+        run: cp build-infection/resources/infection.json5 .
+
+      - name: "Cache Result cache"
+        uses: actions/cache@v4
+        with:
+          path: ./tmp
+          key: "result-cache-v14-${{ matrix.php-version }}-${{ github.run_id }}"
```

Comment:
Why v14?

Reply:
because its just a string and it doesn't matter

Reply:
Yeah but do v1 instead 😊